### DISCUSSION TITLE
MTS ♡ Screenshots

### DISCUSSION THREAD

**Discussion by aecyia** [aecyia/Sakura#32] [Show and tell]
+ Screenshots for my ModTheSims [upload submission](https://modthesims.info/d/673792)
+ Testing World -- Sunlit Tides

![sakura](https://user-images.githubusercontent.com/57078659/218011742-30d375ae-a951-4383-8e43-5f40f3596cce.png)

**Comment by aecyia** [comment]
# v0.0.1

![MainMenu](https://user-images.githubusercontent.com/57078659/218011764-17460076-2aa6-4ad3-aa99-14cd1294039d.jpg)
![EnterGame](https://user-images.githubusercontent.com/57078659/218011769-8e06ce8e-b784-4943-8b4c-55f6c667d44d.jpg)
![EditInCAS](https://user-images.githubusercontent.com/57078659/218011777-f74e534c-a1d2-4a36-ae37-f228b3bf3d27.jpg)
![CreateASim](https://user-images.githubusercontent.com/57078659/218011853-23d1ad7b-c137-47a9-a5aa-28941e5914f3.jpg)
![CreateAHorse](https://user-images.githubusercontent.com/57078659/218011867-ed07c978-a390-45f8-8658-6fa74726d16b.jpg)
![CreateADog](https://user-images.githubusercontent.com/57078659/218011899-51aba334-0d80-46e9-ac46-5d9f9fe6aeca.jpg)
![CreateACat](https://user-images.githubusercontent.com/57078659/218011929-580ab372-5302-4d38-8f8d-0e9d5f8c3de6.jpg)
![CreateAHousehold](https://user-images.githubusercontent.com/57078659/218011799-ce1bb0f9-a26e-489f-a05b-420123bb0679.jpg)
![MainUI](https://user-images.githubusercontent.com/57078659/218012125-19745751-eaa4-4830-8768-c99c4d07ab43.jpg)
![PieMenu](https://user-images.githubusercontent.com/57078659/218012146-90240cdd-df92-45aa-82f0-f3658fb5810d.jpg)
![BuyMode](https://user-images.githubusercontent.com/57078659/218012184-f89deea2-ba79-4f0a-9e7f-36c969510bce.jpg)
![BuildMode](https://user-images.githubusercontent.com/57078659/218012223-9f1613ad-ba7e-4310-ba83-64f5f8cc0bbf.jpg)
![Walls](https://user-images.githubusercontent.com/57078659/218012253-17324dcf-b60a-494e-8ca4-0c4165cfdf7f.jpg)
![CreateAStyle](https://user-images.githubusercontent.com/57078659/218012288-e11c71ef-fed0-463d-9fd4-d441c87c2223.jpg)
![CollectionJournal](https://user-images.githubusercontent.com/57078659/218012377-1e19e330-acbf-47c5-85e0-c2a60d1c483d.jpg)
![ScrapbookMemories](https://user-images.githubusercontent.com/57078659/218012329-54606772-a3ef-48d8-a441-6fa4b539cf96.jpg)
![AdventureJournal](https://user-images.githubusercontent.com/57078659/218012417-3bd7ec86-b489-4f36-ace2-71e9d5c5005b.jpg)

**Comment by aecyia** [comment]
# v0.0.4

![MainMenu](https://user-images.githubusercontent.com/57078659/218019840-50878dd4-446a-4936-bad7-9a773d63377b.jpg)
![EnterGame](https://user-images.githubusercontent.com/57078659/218019847-f50725f4-da3c-4119-9b01-20fd48a0dc58.jpg)
![MainUi_PieMenu](https://user-images.githubusercontent.com/57078659/218019858-f0dffa10-deeb-44e1-84ac-8746d683c65c.jpg)
![PieMenu](https://user-images.githubusercontent.com/57078659/218019865-2ebe1a8c-12c1-4be3-a721-79ba8fc95bc8.jpg)
![TimeAlmanac](https://user-images.githubusercontent.com/57078659/218019935-184f4a89-c407-4ed0-92b7-0b523825c52e.jpg)
![AdventureJournal](https://user-images.githubusercontent.com/57078659/218019936-9b58fbd2-ce9f-4c30-a1e9-4cd435e12bc5.jpg)
![ScrapbookMemories](https://user-images.githubusercontent.com/57078659/218019937-595a0241-90d4-43a2-af0b-74454367a8c6.jpg)
![SkillsJournal](https://user-images.githubusercontent.com/57078659/218019938-990864db-2913-4e91-b512-96a80ffe4e3c.jpg)
![CreateASim](https://user-images.githubusercontent.com/57078659/218019943-0a0471c5-c01e-490c-a784-0aa54bfa60e4.jpg)
![Enter CAS](https://user-images.githubusercontent.com/57078659/218019944-c583366f-5c46-4b66-bd31-8a171903421a.jpg)

**Comment by VMPYRC** [comment]
# v0.0.5

![Scrapbook](https://github.com/aecyia/Sakura/assets/90119648/eb72f3b9-b779-4e6b-8373-f013ea8eef9a)

![AdventureJournal](https://github.com/aecyia/Sakura/assets/90119648/cc05498a-5f64-4b86-af59-b462e12760a0)

**Comment by VMPYRC** [comment]
# v0.0.6 -- #43 

![AdventureJournal](https://github.com/user-attachments/assets/96c7c811-d31d-425e-a5f3-07aa8ad65cb9)
![Scrapbook](https://github.com/user-attachments/assets/90e4f715-9528-43e5-9d5d-b9aaf1c12dea)
![CreateAHousehold](https://github.com/user-attachments/assets/1681d81b-048e-4a03-bfd8-1fc21e7e095c)

**Comment by VMPYRC** [comment]
# v0.0.7 -- #44 

![Scrapbook](https://github.com/user-attachments/assets/ba5385de-eb70-4cc3-9678-f4056a6f443d)
![BlogApp](https://github.com/user-attachments/assets/dded7e40-79b9-4e86-944d-1ebb5ddea2b7)
